docs/specifications/tests/Nameserver-TP/nameserver10.md

```diff
@@ -1,99 +1,165 @@
 # NAMESERVER10: Test for undefined EDNS version
 
+
 ## Test case identifier
 
 **NAMESERVER10**
 
+
+## Table of contents
+
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Special procedural requirements](#Special-procedural-requirements)
+* [Intercase dependencies](#Intercase-dependencies)
+* [Terminology](#terminology)
+
+
 ## Objective
 
 EDNS ([RFC 6891]) is a mechanism to announce capabilities of a DNS
-implementation, and is required by new functionality in DNS such as
-DNSSEC ([RFC 4033][RFC 4033#3], section 3).
+implementation, and is required by new functionality in DNS such as DNSSEC
+([RFC 4033][RFC 4033#section-3], section 3).
 
-[RFC 6891][RFC 6891#6.1.3], section 6.1.3,
-states that if a nameserver has implemented
-EDNS but has not implemented the version level of the request, then it
-MUST respond with RCODE "BADVERS". Only version "0" has been defined for
+[RFC 6891][RFC 6891#section-6.1.3], section 6.1.3, states that if a nameserver
+has implemented EDNS but has not implemented the version level of the request,
+then it MUST respond with RCODE "BADVERS". Only version "0" has been defined for
 EDNS.
 
-Note that RCODE "BADVERS" is an extended RCODE which is calculated from
-the combination of the normal RCODE field in the DNS package header
-([RFC 1035][RFC 1035#4.1.1], section 4.1.1) and the OPT record
-EXTENDED-RCODE field ([RFC 6891][RFC 6891#6.1.3], section 6.1.3). Also see
-[IANA RCODE Registry].
+Note that RCODE "BADVERS" is an extended RCODE which is calculated from the
+combination of the normal RCODE field in the DNS package header
+([RFC 1035][RFC 1035#section-4.1.1], section 4.1.1) and the OPT record
+EXTENDED-RCODE field ([RFC 6891][RFC 6891#section-6.1.3], section 6.1.3). Also
+see [IANA RCODE Registry].
+
 
 ## Scope
 
-It is assumed that *Child Zone* has been tested by [Basic04]. This test
-case will set DEBUG level on messages for non-responsive name servers.
+It is assumed that *Child Zone* has been tested by [Basic04] and [Nameserver02].
+Issues covered by [Basic04] (basic name server issues) or [Nameserver02] (basic
+EDNS issues) will not result in messages from this test case.
```

Comment:
The first sentence here could be removed. AFAICT it doesn't add anything over what's being said in the second sentence.

Reply:
Probably not. And it is not clear what the "assumption" is. I'll remove it.